### PR TITLE
[5.1] Added seeJsonKeys and dontSeeJsonKeys to CrawlerTrait

### PR DESCRIPTION
**Pull request by samlev**
Added methods to the CrawlerTrait which allow you to search
for json keys without caring about content.

This is useful if returned data is long, or the exact value doesn't
really matter.

### CODE REVIEW

**Comment by samlev** [comment]
Bump for CI (fixed issues and re-pushed)

File: src/Illuminate/Foundation/Testing/CrawlerTrait.php [hunk 1]
```diff
@@ -528,6 +528,82 @@ public function seeJson(array $data = null, $negate = false)
...
     /**
+     * Assert that the response contains the given JSON keys.
+     *
+     * Keys can be formatted with dot notation, or as nested arrays:
+     *   $keys = [ 'key1', 'key2' => ['subkey1'], 'key3.subkey2' ]
+     *
+     * Would match this json reponse:
+     *   {"key1": "data", "key2": {"subkey1": "data"}, "key3": {"subkey2": "data"}}
+     *
+     * @param  array|null  $data A list of JSON keys to look for
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Testing/CrawlerTrait.php]: Please don't document parameters inline like this. If extra clarification is needed, please include that in the "long description".

File: src/Illuminate/Foundation/Testing/CrawlerTrait.php [hunk 2]
```diff
@@ -528,6 +528,82 @@ public function seeJson(array $data = null, $negate = false)
     }
 
     /**
+     * Assert that the response contains the given JSON keys.
+     *
+     * Keys can be formatted with dot notation, or as nested arrays:
+     *   $keys = [ 'key1', 'key2' => ['subkey1'], 'key3.subkey2' ]
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Testing/CrawlerTrait.php]: cs

**Comment by GrahamCampbell** [comment]
Why did you close this?

**Comment by samlev** [comment]
I closed it to get CI services to notice the updated commit

**Comment by samlev** [comment]
For some reason github won't let me re-open it, so I made a new one.

**Pull request closed without merging**